config/env.ini

```diff
@@ -64,7 +64,7 @@ UPX_EXEC="${PKG_ROOT_PATH}\bin\upx.exe"
 ; phpmicro patches, for more info, see: https://github.com/easysoft/phpmicro/tree/master/patches
 SPC_MICRO_PATCHES=static_extensions_win32,cli_checks,disable_huge_page,vcruntime140,win32,zend_stream,cli_static
 ; Windows static linking system libs
-SPC_WINDOWS_EXEC_LIBS="ws2_32.lib shell32.lib"
+SPC_EXTRA_LIBS=""
```

Comment:
Not necessary here, can also be removed from macos. An empty definition can be casted to empty string.